### ISSUE TITLE
block highlight

### ISSUE THREAD

**Issue by ayoubelmhamdi** [charmbracelet/glow#354]
could anyone add a bock highlight for this excellent project, be like github:

- glow  

![glow](https://user-images.githubusercontent.com/34358145/177046385-c66e9837-7d24-4569-ab40-6d2056334d7f.png)

- github  

![github2](https://user-images.githubusercontent.com/34358145/177046423-a2369920-b9e8-4de2-bd3f-7f3d8ca5f307.png)

**Comment by WieeRd** [comment]
I agree, an option to make codeblock backgrounds darker/lighter would be cool.
